tutorxqueue/plugin.py

```diff
@@ -256,22 +258,36 @@ def request(self, endpoint, method="GET", data=None, params=None):
     )
 ):
     with open(path, encoding="utf-8") as patch_file:
-        tutor_hooks.Filters.ENV_PATCHES.add_item((os.path.basename(path), patch_file.read()))
+        tutor_hooks.Filters.ENV_PATCHES.add_item(
+            (os.path.basename(path), patch_file.read())
+        )
 
 # Add cli commands filter
 tutor_hooks.Filters.CLI_COMMANDS.add_item(command)
 
 # Add configuration entries
 tutor_hooks.Filters.CONFIG_DEFAULTS.add_items(
-    [
-        (f"XQUEUE_{key}", value)
-        for key, value in config.get("defaults", {}).items()
-    ]
+    [(f"XQUEUE_{key}", value) for key, value in config.get("defaults", {}).items()]
 )
 tutor_hooks.Filters.CONFIG_UNIQUE.add_items(
-    [
-        (f"XQUEUE_{key}", value)
-        for key, value in config.get("unique", {}).items()
-    ]
+    [(f"XQUEUE_{key}", value) for key, value in config.get("unique", {}).items()]
+)
+tutor_hooks.Filters.CONFIG_OVERRIDES.add_items(
+    list(config.get("overrides", {}).items())
 )
-tutor_hooks.Filters.CONFIG_OVERRIDES.add_items(list(config.get("overrides", {}).items()))
+
+
+########################################
+# Xqueue Public Host
+########################################
+
+
+@tutor_hooks.Filters.APP_PUBLIC_HOSTS.add()
+def _discovery_public_hosts(
+    hosts: list[str], context_name: t.Literal["local", "dev"]
+) -> list[str]:
+    if context_name == "dev":
+        hosts += ["discovery.{{ LMS_HOST }}:8000"]
```

Comment:
```suggestion
        hosts += ["xqueue.{{ LMS_HOST }}:8000"]
```
@irtazaakram the same fix should be applied 2 lines below

Reply:
@jfavellar90 Done 👍🏻